packages/content-cta/stories/examples.stories.js

```diff
@@ -1,51 +0,0 @@
-import React from 'react'
```

Comment:
There was some old Storybook content here which is no longer relevant, so removed it 🔪